Sming/SmingCore/Network/WebsocketClient.cpp

```diff
@@ -99,10 +100,11 @@ void WebsocketClient::onError(err_t err)
 
 bool WebsocketClient::_verifyKey(char* buf, int size)
 {
-	String dd = String(buf);
-	uint8_t s = dd.indexOf("Sec-WebSocket-Accept: ");
+	String keyHeader("Sec-WebSocket-Accept: ");
+	String dd(buf);
+	uint8_t s = dd.indexOfIgnoreCase(keyHeader);
 	uint8_t t = dd.indexOf("\r\n", s);
```

Comment:
For `s` and `t` - add checks if the return value is `-1` and return false in that case.